test/e2e/framework/framework.go

```diff
@@ -176,6 +179,25 @@ func getClientRepoConfig(src *restclient.Config) (dst *clientreporestclient.Conf
 	return dst
 }
 
+func (f *Framework) whoAmI() (*rbacv1beta1.Subject, error) {
+	if ProviderIs("gke") {
+		out, err := exec.Command("gcloud", "auth", "list", "--filter=status:ACTIVE", "--format=value(account)").CombinedOutput()
+		Logf("WRF:whoAmi error %v", err)
+		Logf("WRF:whoAmi %s", string(out))
+		if err != nil {
+			return nil, err
+		}
+		name := string(out)
+		if strings.HasSuffix(name, ".iam.gserviceaccount.com") {
```

Comment:
Removed all "dead" code. It will be coming back once I get a chance to fix the TODO. Will make sure this is correct once I can test with a service account.